src/Soil/soil_hydrology_parameterizations.jl

```diff
@@ -128,6 +173,103 @@ function hydraulic_conductivity(K_sat::FT, m::FT, S::FT) where {FT}
 end
 
 
+"""
+     matric_potential(cm::BrooksCorey{FT}, S::FT) where {FT}
```

Comment:
Maybe we should add documentation somewhere explaining which functions we expect to be implemented for each closure model concrete type. Right above `AbstractSoilHydrologyClosure` might be a good place for that.

Reply:
good idea!